pyproject.toml

```diff
@@ -37,7 +37,11 @@ dataset_creation = [
         "PyTDC",
         "rdkit-pypi",
         "ruamel.yaml",
-        "modin[all]"
+        "modin[all]",
+        "selfies",
+        "deepsmiles",
+        "pubchempy",
+        #"tucan@git+https://github.com/TUCAN-nest/TUCAN.git"  # the current version has bugs due to the type checking, maybe this is due to our python version?
```

Comment:
yes, it only works in some python versions. perhaps let's ignore tucan for now.

Reply:
yeah, it would be easy to add if we fork it but we can do that also later, the code is prepared so that it should work out of the box.